docs/src/content/pages/collections.mdoc

```diff
@@ -37,29 +37,84 @@ export default config({
 
 ### Columns
 
-`columns` — show additional fields in the collection list view. 
+By default, only the “slug” of each entry is displayed in the collection table. You can show additional fields by passing a `columns` option.
```

Comment:
> By default, only the “slug” of each entry is displayed in the collection table. You can show additional fields by passing a `columns` option.

This is no longer true with this PR, `columns` now removes the slug instead of just adding other columns. I don't think we should land this without preserving the ability to show the slug and other fields.